flixel/util/FlxTimer.hx

```diff
@@ -156,16 +156,23 @@ class FlxTimer implements IFlxDestroyable
 		{
 			_timeCounter -= time;
 			_loopsCounter++;
-			
-			if (onComplete != null)
-			{
-				onComplete(this);
-			}
-			
-			if (loops > 0 && (_loopsCounter >= loops))
-			{
-				cancel();
-			}
+		}
+	}
+	
+	/**
+	 * Called by timer manager when timer finishes loop.
+	 */
+	@:allow(flixel.util.FlxTimerManager)
+	private function onLoop():Void
```

Comment:
Should be `onLoopFinished()`, which I guess also makes the doc comment obsolete. :)